react-components/src/components/Reveal3DResources/Reveal3DResources.tsx

```diff
@@ -67,16 +51,12 @@ export const Reveal3DResources = ({
     getTypedModels(resources, viewer).then(setReveal3DModels).catch(console.error);
   }, [resources, viewer]);
 
-  const modelsStyling = useCalculateModelsStyling(reveal3DModels, styling, fdmAssetMappingConfig);
-
-  useEffect(() => {
-    setReveal3DModelsStyling(modelsStyling);
-  }, [modelsStyling]);
+  const reveal3DModelsStyling = useCalculateModelsStyling(reveal3DModels, instanceStyling ?? []);
```

Comment:
This variable doesn't seem to be used anywhere, is it necessary?

Reply:
It is used further down and passed to appropriate models.